packages/react/.eslintrc.js

```diff
@@ -65,12 +65,10 @@ module.exports = {
     {
       files: [
         // TODO Uncomment the below lines on an individual basis as the updates to pass linting are completed.
-        // If needed, these can be broken down further, e.g. 'src/components/Geo/**/*'
-        'src/components/Geo/**/*',
-        'src/components/ThemeProvider/**/*',
+        // If needed, these can be broken down further, e.g. 'src/primitives/Alert/**/*'
+        'src/components/**/*',
```

Comment:
This will be breaking change in the staging repo. Can we keep it scoped to per component?